pom.xml

```diff
@@ -151,19 +151,19 @@
             <dependency>
                 <groupId>io.projectreactor</groupId>
                 <artifactId>reactor-test</artifactId>
-                <version>3.1.8.RELEASE</version>
+                <version>3.1.9.RELEASE</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-core</artifactId>
-                <version>2.18.3</version>
+                <version>2.22.0</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-junit-jupiter</artifactId>
-                <version>2.18.3</version>
+                <version>2.20.1</version>
```

Comment:
The latest version for ```mockito-junit-jupiter``` is ```2.22.0```. But the latest version seems to have bug. When I updated the version  to ```2.22.0```, I got the below exception. They have raised the issue in junit forum.

```
https://github.com/junit-team/junit5/issues/833
```

```
Tests run: 6, Failures: 0, Errors: 5, Skipped: 0, Time elapsed: 0.318 sec <<< FAILURE! - in com.vmware.connectors.aws.cert.AwsCertControllerMockedTest
testApprove{String, int}[1]  Time elapsed: 0.022 sec  <<< ERROR!
org.junit.jupiter.api.extension.ParameterResolutionException: Failed to resolve parameter [java.lang.String approvalUrl] in executable [void com.vmware.connectors.aws.cert.AwsCertControllerMockedTest.testApprove(java.lang.String,int)]
Caused by: java.lang.AbstractMethodError

```

Reply:
Please try updating the maven-surefire-plugin to 2.22.0, removing the dependencies from it, then adding junit-jupiter-engine under dependencyManagement. See https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html